lib/Sema/ConstraintSystem.cpp

```diff
@@ -5689,3 +5689,18 @@ TypeVarBindingProducer::getDefaultBinding(Constraint *constraint) const {
              ? binding.withType(OptionalType::get(type))
              : binding;
 }
+
+bool constraints::isOperatorDisjunction(Constraint *disjunction) {
```

Comment:
This looks really similar to the code in `isOperatorBindOverload` in CSSolver.cpp. Could this new function subsume the old one?

Reply:
heh, I suspected something like that but just couldn't find it...

Reply:
Fixed by https://github.com/apple/swift/pull/36911.